WordPress/openverse
# Add (or update) robots.txt and ai.txt to block AI crawlers

## Fixes



Fixes #3900 by @zackkrida 

## Description


Following the advice of the blog post Zack linked (https://neil-clarke.com/block-the-bots-that-feed-ai-models-by-scraping-your-website/), I've added or updated robots.txt and added ai.txt accordingly, to block all crawlers.

As dicussed in the blog post and in https://github.com/WordPress/openverse/discussions/3916, this isn't a perfect solution, and there are malicious, bad-faith actors who ignore robots.txt. We'll address those using other tools at our disposal.

Docs preview pages:
- https://docs.openverse.org/_preview/4077/robots.txt
- https://docs.openverse.org/_preview/4077/ai.txt

## Testing Instructions

Visit `/robots.txt` and `/ai.txt` on each of:
- the API, run `just api/up`. Note that `robots.txt` is no longer incorrectly redirected to `robots.txt/` with a trailing slash. semantically the trailing slash makes no sense for a document resource, and in the worst case scenario is plausible deniability for not-entirely-bad-but-still-sketchy actors who won't follow a redirect on `robots.txt`.
- the documentation site, run `just documentation/live` and visit each
- the frontend, run `just p frontend dev` and check each

## Checklist


- [N/A] I added or updated tests for the changes I made (if applicable).
- [N/A] I added or updated documentation (if applicable).
- [N/A] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
